gurubu-client/src/app/styles/room/nickname-form.scss

```diff
@@ -76,22 +133,20 @@
   }
   &__button {
```

Comment:
LGTM! I resolved other comments just ı saw this can we add font-family inter into `__button` after that we can merge